### PR TITLE
XDR Malware Enrichment - hotfix for usernames (split)

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-27200

## Description
Updated the `Cortex XDR Malware - Incident Enrichment` playbook to split usernames from domains before enriching the usernames, in order to allow usernames to be enriched in integrations that don't support domains, in scenarios where "domain\usernames" are ingested from Cortex XDR.

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/5_1_6.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR Malware - Incident Enrichment
+
+- Added tasks to extract usernames from domain usernames in order to allow enrichment with both integrations that domain usernames and integrations that allow only username enrichment.
```
Comment (tomer-pan) [inline Packs/CortexXDR/ReleaseNotes/5_1_6.md:6]: I think you are missing the word "allows" after the word "that" in this part:
"...with both integrations that domain usernames and integrations..."

**Review by tomer-pan — APPROVED** [review]
Looks good, please address my 2 comments before merging.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml [hunk 2]
```diff
@@ -1283,6 +1290,115 @@ tasks:
...
+          filters:
+          - - operator: containsGeneral
+              left:
+                value:
+                  simple: incident.users
+                iscontext: true
+              right:
+                value:
+                  simple: \
+          transformers:
```
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml:1375]: Unique is there, appears as "Uniq" under my changes :)

File: Packs/CortexXDR/ReleaseNotes/5_1_6.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR Malware - Incident Enrichment
+
+- Added tasks to extract usernames from domain usernames in order to allow enrichment with both integrations that accept domain usernames and integrations that accept only usernames for enrichment.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/5_1_6.md:6]:
```suggestion
Added tasks to extract usernames from domain usernames in order to allow enrichment with both integrations that accept domain usernames and integrations that accept only usernames for enrichment.
```

**Comment by ShirleyDenkberg** [comment]
@tomer-pan @AdiPeret Doc review completed.

**Pull request merged**